docs/source/install/install.rst

```diff
@@ -123,6 +123,18 @@ Additional CMake options can be passed via individual environment variables, whi
 
    .. image:: cmake.svg
 
+Using the Julia Package
+-----------------------
+
+A package for openPMD-api is available via the Julia Package Registry:
+
+.. code-block:: julia
+
+    # We need Julia 1.7 or newer
+    using("Pkg")
+    Pkg.add("openPMD")
```

Comment:
That sounds fine. I would point to `openPMD.jl` as an existing package that uses these bindings, so that people are not put off by the experimental status – it's not openPMD that is experimental, it's only the way things are currently glued together.